extensions/database/src/com/google/refine/extension/database/DatabaseModuleImpl.java

```diff
@@ -96,6 +96,22 @@ private static int getBatchSize(String propertyName, int defaultValue) {
         return defaultValue;
     }
 
+    /**
+     * @deprecated use {@link #getCreateBatchSize()} instead.
+     */
+    @Deprecated(since = "3.10")
+    public static String getImportCreateBatchSize() {
+        return String.valueOf(getCreateBatchSize());
+    }
+
+    /**
+     * @deprecated use {@link #getPreviewBatchSize()} instead.
+     */
+    @Deprecated(since = "3.10")
+    public static String getImportPreviewBatchSize() {
+        return String.valueOf(getPreviewBatchSize());
+    }
+
```

Comment:
It looks like these were just added in October in 8c769bdcad345f23aef92c7e09b041abc7308f6e, so I think we can just delete them without deprecation.
```suggestion
```

Reply:
Before the linked commit you had two methods with signatures
* `public static String getImportCreateBatchSize()`
* `public static String getImportPreviewBatchSize()`

-> these methods were in db extension since addition to this repo (8 years ago).


And after the linked commit from October the methods are replaced by
* `public static int getCreateBatchSize()`
* `public static int getPreviewBatchSize()`

=> name and return type changed, 
old methods point to the new methods in their deprecation message